src/mame/audio/8080bw.h

```diff
@@ -0,0 +1,44 @@
+// license:BSD-3-Clause
```

Comment:
This file hasn't been added to the build scripts.

Reply:
Oh, sorry. I didn't take in account the build scripts. I compiled from command line with make and it compiled fine. 
I'm gonna look at the commit you've made in the source tree to understant what I missed.

Reply:
Yeah, it will build if you don't add a header to the scripts, but for people using features like building with Visual Studio the file won't appear in the IDE project, and dependencies may not be computed correctly.